### ISSUE TITLE
15459849: The Accessibility Inspector in the iPhone Simulator should be an external window

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#4211]
#### Description

Summary:
The Accessibility Inspector in the Simulator covers part of the simulated application. With long labels the content does not fit. Also the number of notifications it can display is limited, and there's no timestamp associated.

Steps to Reproduce:
1. Enable Settings/General/Accessibility/Accessibility Inspector
2. Open any app with decent accessibility support, and select a control. An example would be Photos in iOS7, assuming you have no photos, tap on "You can sync photos and videos".

Actual Results:
http://d.pr/i/AFBH

Expected Results:
A top level window would avoid all this problems
## 

Product Version: Xcode 5.0.2
Created: 2013-11-13T17:57:46.077880
Originated: 2013-11-13T14:57:00
Open Radar Link: http://www.openradar.me/15459849
